tests/baselines/reference/user/puppeteer.log

```diff
@@ -1,6 +1,16 @@
 Exit Code: 1
 Standard output:
 lib/Accessibility.js(118,15): error TS2503: Cannot find namespace 'Protocol'.
+lib/Accessibility.js(310,7): error TS2322: Type 'string | number | boolean' is not assignable to type 'never'.
```

Comment:
```ts
node[userStringProperty] = properties.get(userStringProperty) :
    SerialisedAXNode[keyof SerialisedAXNode] <== number | string | boolean
```

where

```ts
keyof SerialisedAXNode : string | number | boolean | SerialisedAXNode[]
```

The difference is *probably* that now

```ts
SerialisedAXNode[keyof SerialisedAXNode] : never
```

Reply:
Yeah, this is an intended result of https://github.com/Microsoft/TypeScript/pull/30769 - since there's no generic tying `userStringProperty` to the result of `properties.get(userStringProperty)`, we don't know they you're assigning the value corresponding to the selected key to the correct key'd member. Unfortunately, you need to write a subtype of `Map`'s type to do that (otherwise `Map.get` has no method type parameter to be generic on).